### ISSUE TITLE
High processor usage at solving non html url

### ISSUE THREAD

**Issue by MrMaxie** [yue/yue#48]
When I tried to resolve url with extension other than .html - usage of processor drasticly increments. From about 0,5% of usage to about 26% at resolving that non-html url. And nothing change over time. Process still have that high processor usage. Second try of it make usage about 50%, third on about 75%.

**Reproducing**
To be able to reproduce this bug just use the sample code in the form of "Mugan", add two files `test.html` and `test.css` and make two anchors, one for each of them:
```html
<a href='mugan://app/test.html'>test.html</a>
<a href='mugan://app/test.css'>test.css</a>
```
First of them works great. But second makes process usage higher for nothing.

**Comment by zcbenz** [comment]
Which operating system are you using?

**Comment by MrMaxie** [comment]
That issue happens on Windows 10

**Comment by zcbenz** [comment]
Is it possible for you to provide a minimal app that can reproduce this problem? I followed your description but could not reproduce.

**Comment by MrMaxie** [comment]
Okay, I can just step-by-step describe it, because it's really only few steps using Muban (not Mugan, sorry for previous typos). Creating separated demo IMHO it's redundant.

- Clone Muban
- `npm install` && `npm start` for test
- Change `index.html` into:
```html
<!DOCTYPE html>
<html>
<body>
<head>
<title>test</title>
</head>
<body>
  <a href='muban://app/test.html'>test.html</a>
  <a href='muban://app/test.css'>test.css</a>
</body>
</html>
```
- Add files: `test.html` and `test.css` inside `/app` directory. They can be with any content, either empty.
- `npm run-script build` && `npm start`
- Resolving url for `test.html`:
![image](https://user-images.githubusercontent.com/3857836/51749451-af10f300-20af-11e9-86e5-28b59c6f6c7b.png)
- First try to resolve `test.css`:
![image](https://user-images.githubusercontent.com/3857836/51749498-d2d43900-20af-11e9-8d6f-a18ceb7f833b.png)
- Second try to resolve `test.css`:
![image](https://user-images.githubusercontent.com/3857836/51749528-e384af00-20af-11e9-83d9-7f2bc299c368.png)
- Third try to resolve `test.css`:
![image](https://user-images.githubusercontent.com/3857836/51749556-f6977f00-20af-11e9-91da-1fca45d9c087.png)

Ps sorry for polish names of columns. They are in order: "Processor", "RAM", "Disk", "Network".

Tell me if there is way to give you more informations about that, i'll help you as well as i can because i love concept of this project.

**Comment by zcbenz** [comment]
Thanks for the detail steps, I can reproduce the problem with it.

**Comment by zcbenz** [comment]
Fixed in 0.6.3.
